flapjack/ext.py

```diff
@@ -112,7 +112,13 @@ def _generate_manifest():
 
         config_opts = config.module_extra_config_opts(m['name'])
         if config_opts:
```

Comment:
I would rename `config_opts` to `extra_config_opts`, for faster understanding, but otherwise looks good!

Reply:
Fixed, thanks!